src/components/card/index.tsx

```diff
@@ -231,8 +231,8 @@ class Card extends PureComponent<PropTypes, State> {
         {!selectionOptions.hideIndicator && (
           <View style={[this.styles.selectedIndicator, {backgroundColor: selectionColor}]}>
             <Icon
-              style={this.styles.selectedIcon}
               source={_.get(selectionOptions, 'icon', DEFAULT_SELECTION_PROPS.icon)}
+              tintColor={_.get(selectionOptions, 'iconColor', DEFAULT_SELECTION_PROPS.iconColor)}
```

Comment:
While we're here, maybe worth removing `_.get` usage and move to optional chaining

Reply:
Done :)